ocis/cmd/ocis/main.go

```diff
@@ -3,10 +3,16 @@ package main
 import (
 	"os"
 
+	"github.com/KimMachineGun/automemlimit/memlimit"
 	"github.com/owncloud/ocis/v2/ocis/pkg/command"
 )
 
 func main() {
+
+	if _, present := os.LookupEnv("GOMEMLIMIT"); !present {
+		memlimit.SetGoMemLimit(0.9)
```

Comment:
Is there a reason why you aren't using `memlimit.SetGoMemliimtWithEnv()`. That would allow to turn the whole thing off with `AUTOMEMLIMIT=off`